plugins/git/gitblame.nix

```diff
@@ -6,66 +6,133 @@
+      You can view an up-to-date full list of options in the
+      [Neovim documentation](https://neovim.io/doc/user/api.html#nvim_buf_set_extmark()).
+
+      **Warning:** overwriting `id` and `virt_text` will break the plugin behavior.
+    '';
+
+    display_virtual_text = nixvim.defaultNullOpts.mkBool true ''
+      If the blame message should be displayed as virtual text.
+      You may want to disable this if you display the blame message in statusline.
+    '';
+
+    ignored_filetypes = nixvim.defaultNullOpts.mkListOf types.str [ ] ''
+      A list of filetypes for which gitblame information will not be displayed.
+    '';
+
+    delay = nixvim.defaultNullOpts.mkUnsignedInt 250 ''
+      The delay in milliseconds after which the blame info will be displayed.
+    '';
+
+    virtual_text_column = nixvim.defaultNullOpts.mkUnsignedInt null ''
+      Have the blame message start at a given column instead of EOL.
+      If the current line is longer than the specified column value, the blame message will default
+      to being displayed at EOL.
+    '';
+
+    use_blame_commit_file_urls = nixvim.defaultNullOpts.mkBool false ''
+      By default the commands `GitBlameOpenFileURL` and `GitBlameCopyFileURL` open the current file
+      at latest branch commit.
+      If you would like to open these files at the latest blame commit (in other words, the commit
+      marked by the blame), set this to true.
+      For ranges, the blame selected will be the most recent blame from the range.
+    '';
+
+    schedule_event = nixvim.defaultNullOpts.mkStr "CursorMoved" ''
+      If you are experiencing poor performance (e.g. in particularly large projects) you can use
+      `CursorHold` instead of the default `CursorMoved` autocommand to limit the frequency of events
+      being run.
+    '';
+
+    clear_event = nixvim.defaultNullOpts.mkStr "CursorMovedI" ''
+      If you are experiencing poor performance (e.g. in particularly large projects) you can use
+      `CursorHoldI` instead of the default `CursorMovedI` autocommand to limit the frequency of
+      events being run.
+    '';
+
+    clipboard_register = nixvim.defaultNullOpts.mkStr "+" ''
+      By default the `:GitBlameCopySHA`, `:GitBlameCopyFileURL` and `:GitBlameCopyCommitURL`
+      commands use the `+` register.
+      Set this value if you would like to use a different register (such as `*`).
+    '';
+  };
+
+  settingsExample = {
+    message_template = "<summary> • <date> • <author>";
+    date_format = "%r";
+    message_when_not_committed = "Oh please, commit this !";
+    highlight_group = "Question";
+    set_extmark_options.priority = 7;
+    display_virtual_text = false;
+    ignored_filetypes = [
+      "lua"
+      "c"
+    ];
+    delay = 1000;
+    virtual_text_column = 80;
+    use_blame_commit_file_urls = true;
   };
 
-  config =
-    let
-      setupOptions = {
-        enabled = cfg.enable;
```

Comment:
Looks like we used to set `settings.enabled` by default. Is there a way we can warn that this is no longer the case?

Reply:
Anyway, the default behavior is still `true`. Now the user has the option to explicitly disable it.